### PR TITLE
Extra classes in option outerClass

### LINKED ISSUE #31: Extra classes with option "outerClass"
With our popup, we only add one class for popup outer `.easy-popup`. Adding more classes to the custom style for the popups seems insufficient. Can we do that?

### PR DESCRIPTION
**Pull request by tung13041996**
This PR resolves this issue: https://github.com/viivue/easy-popup/issues/31

### CODE REVIEW

**Review by phucbm — CHANGES_REQUESTED** [review]
Please also update the docs with the new changes about `outerClass`.

**Comment by tung13041996** [comment]
@phucbm 
I updated it.

File: dev/md/options.md [hunk 1]
```diff
@@ -10,7 +10,7 @@
 | `mobileBreakpoint`     | number                 | `768`             | Activate mobile layout when the screen size is <=768px                                                                   |
 | `closeButtonInnerText` | string                 | `svg/icon`        | Custom `innerText` of the close button                                                                                   |
 | `triggerSelector`      | CSS selector           | `""`              | Click on this trigger will also toggle the popup                                                                         |
-| `outerClass`           | string                 | `""`              | Extra class to popup outer `.easy-popup`                                                                                 |
+| `outerClass`           | string                 | `""`              | Extra classes to popup outer `.easy-popup`                                                                               |
```
Comment (phucbm) [inline dev/md/options.md]: Add this sentence:

Use white space for multiple classes, e.g. `"class1 class2"`.

Comment (tung13041996) [reply dev/md/options.md]: I updated. Can you check it again? Thanks.

**Review by phucbm — APPROVED** [review]

**Pull request merged**